src/react/components/forms/CreateDealForm.tsx

```diff
@@ -262,7 +262,7 @@ export const CreateDealForm = (props: Props) => {
 				<br />
 				<label className="stake-input-label">
 					Financing fee %
-					<p>The percentage on top of the principal that needs to be repaid as interest (APR)</p>
+					<p>The percentage on top of the principal that needs to be repaid as interest (APY)</p>
```

Comment:
it's probably my lack of finance knowledge but why is it APY instead of APR here? I don't see where compounding is relevant here as the financingfee is solely dependent on the principal, not principal + some value that needs to be compounded (which is what makes APY relevant as far as I know)